### ISSUE TITLE
Network for DevOps engineers

### ISSUE THREAD

**Issue by mamedxanli** [goupaz/0-to-hero#22]
I would recommend the following for reading and video for developers moving to DevOps role :
http://www.ciscopress.com/markets/detail.asp?st=44711
There are video courses and books on this link which should suffice for general understanding of networking. Don't be worried about learning something vendor specific - Cisco is a standard developer for the industry. 

Best free emulator/simulator software (Might require some effort to get it properly configured, up and running. But, do not get stuck into how the software works. It has almost unlimited number of features and there is no point to waste time on perfecting the GNS3 itself. Its just a tool):
https://www.gns3.com/

Free labs and challenges:
https://gns3vault.com/labs

**Comment by NitrousOxyde** [comment]
IMHO eve-ng has more features and better usability that GNS3. Additionally, it supports way more platforms than GNS3 does. Personally, I've been using eve-ng for last 2 years while preparing for CCIE/JNCIE, works perfect.

For ones who are interested - http://www.eve-ng.net/

**Comment by mamedxanli** [comment]
> IMHO eve-ng has more features and better usability that GNS3. Additionally, it supports way more platforms than GNS3 does. Personally, I've been using eve-ng for last 2 years while preparing for CCIE/JNCIE, works perfect.
> 
> For ones who are interested - http://www.eve-ng.net/

Yes, there are many other tools out there with even more features than GNS3. I recommended GNS3 as a tool for CCNA level networker with primary focus on dev/sysengineer/automation/devops rather than network itself. I think GNS3 would do for that matter.

**Comment by NitrousOxyde** [comment]
> 
> 
> > IMHO eve-ng has more features and better usability that GNS3. Additionally, it supports way more platforms than GNS3 does. Personally, I've been using eve-ng for last 2 years while preparing for CCIE/JNCIE, works perfect.
> > For ones who are interested - http://www.eve-ng.net/
> 
> Yes, there are many other tools out there with even more features than GNS3. I recommended GNS3 as a tool for CCNA level networker with primary focus on dev/sysengineer/automation/devops rather than network itself. I think GNS3 would do for that matter.

Agree, for CCNA level, or even CCNP level gns3 might be enough.
